app/models/health/careplan.rb

```diff
@@ -269,8 +296,12 @@ def active?
       completed? && expires_on >= Date.current
     end
 
+    CP2_DATE = '2023-04-01'.to_date.freeze
     def completed?
-      provider_signed_on && patient_signed_on
+      cp1 = provider_signed_on && patient_signed_on && [provider_signed_on, patient_signed_on].max < CP2_DATE
+      cp2 = patient_signed_on >= CP2_DATE && rn_approval?
```

Comment:
Where is `rn_approval?` coming from.  I can't find it.  Also, does it guarantee `patient_signed_on` is present?